airflow/providers/amazon/aws/operators/ecs.py

```diff
@@ -39,7 +39,7 @@
     EcsTaskDefinitionStates,
     should_retry_eni,
 )
-from airflow.providers.amazon.aws.sensors.ecs import EcsClusterStateSensor, EcsTaskDefinitionStateSensor
+from airflow.utils.helpers import prune_dict
```

Comment:
Nice.  I have a bunch of EMR waiters I have been workigb on converting over but put on pause to verify how they behave if `None`  is passed in, but this would solve that nncely.  I'll get the EMR ones up this week and tag you in them.  :+1: